src/get-options-overrides.ts

```diff
@@ -0,0 +1,11 @@
+import {CompilerOptions, ModuleKind} from "typescript";
+
+export function getOptionsOverrides(): CompilerOptions {
+	return {
+		module: ModuleKind.ES2015,
+		noEmitHelpers: true,
+		importHelpers: true,
+		noResolve: false,
+		outDir: process.cwd(),
```

Comment:
Yes, values provided in rollup should override tsconfig, but in their absence we should use tsconfig values. This will make more setups just work (and will not break existing setups).